pkgs/net-lib/net/server.rkt

```diff
@@ -1,15 +1,17 @@
 #lang racket/base
 
 (require racket/contract
-         racket/tcp)
+         racket/tcp
+         (only-in racket/unix-socket
```

Comment:
Since `unix-socket-lib` isn't (and probably shouldn't) be a dependency of `net-lib`, it's probably best to leave this import out. The `evt?` contract should be enough to match both channels and a unix socket listener.